inventory/views.py

```diff
@@ -1,72 +1,139 @@
 
 
 def index(request):
     categories = Category.objects.all()
-    context = {'categories': categories}
-    return render(request, 'inventory/index.html', context)
+    context = {"categories": categories}
+    return render(request, "inventory/index.html", context)
 
 
-def Display_Category(request, category_id):
+def display_category(request, category_id):
     category = Category.objects.get(pk=category_id)
     header = "Currently Viewing " + category.name
     context = {
-        'items': category.items.all(),
-        'name': category.name,
-        'header': header,
-        'category_id':category_id
+        "items": category.items.all().select_related("electricalitem"),
+        "name": category.name,
+        "header": header,
+        "category_id": category_id,
+        "category": category,
     }
-    return render(request, 'inventory/DisplayCategory.html', context)
+    return render(request, "inventory/DisplayCategory.html", context)
 
 
-def Add_Item(request, category_id):
+def add_item(request, category_id):
     if request.method == "POST":
-        form = AddItemForm(request.POST,request.FILES)
-        print (form.errors)
+        form = AddItemForm(request.POST, request.FILES)
+        print(form.errors)
         if form.is_valid():
-            member = form.cleaned_data['member']
-            title = form.cleaned_data['title']
-            description = form.cleaned_data['description']
-            width = form.cleaned_data['width']
-            height = form.cleaned_data['height']
-            length = form.cleaned_data['length']
-            voltage = form.cleaned_data['voltage']
-            wattage = form.cleaned_data['wattage']
-            quantity = form.cleaned_data['quantity']
-            unit_price = form.cleaned_data['unit_price']
-            picture = form.cleaned_data['picture']
-            #item = Item(member=member,title=title,description=description,width=width,height=height,length=length,quantity=quantity,unit_price=unit_price,picture=picture)
-            #item.save()
-            item = Item.objects.create(member=member,title=title,description=description,width=width,height=height,length=length,voltage=voltage,wattage=wattage,quantity=quantity,unit_price=unit_price,picture=picture)
+            member = form.cleaned_data["member"]
+            title = form.cleaned_data["title"]
+            description = form.cleaned_data["description"]
+            width = form.cleaned_data["width"]
+            height = form.cleaned_data["height"]
+            length = form.cleaned_data["length"]
+            quantity = form.cleaned_data["quantity"]
+            unit_price = form.cleaned_data["unit_price"]
+            picture = form.cleaned_data["picture"]
+            # item = Item(member=member,title=title,description=description,width=width,height=height,length=length,quantity=quantity,unit_price=unit_price,picture=picture)
+            # item.save()
+            item = Item.objects.create(
```

Comment:
why are you manually creating an item. That's the whole point of using forms
check the other code in membership app for reference